packages/core/src/types/config.ts

```diff
@@ -94,6 +94,12 @@ export type Dts =
        * @see {@link https://rslib.rs/config/lib/dts#dtsalias}
        */
       alias?: Record<string, string>;
+      /**
+       * [Experimental] Whether to generate declaration files with `tsgo`.
```

Comment:
```suggestion
       * Whether to generate declaration files with `tsgo`.
       * @experimental
```

https://tsdoc.org/pages/tags/experimental/